src/Kiota.Builder/Refiners/RubyRefiner.cs

```diff
@@ -21,8 +19,9 @@ public override void Refine(CodeNamespace generatedCode)
                                                 }, _configuration.UsesBackingStore, true);
             ReplaceReservedNames(generatedCode, new RubyReservedNamesProvider(), x => $"{x}_escaped");
             ReplaceRelativeImportsByImportPath(generatedCode, '.');
+            AddNamespaceModuleImports(generatedCode , _configuration.ClientNamespaceName);
             FixReferencesToEntityType(generatedCode);
-            FixInheritedEntityType(generatedCode, null, "Graphrubyv4::Utilities::Users::");
+            FixInheritedEntityType(generatedCode, null, _configuration.ClientNamespaceName.NormalizeNameSpaceName("::") + "::Users::");
```

Comment:
users should not be hard coded here, nothing guarantees we'll have users in the api description